CONTRIBUTING.md

```diff
@@ -24,144 +24,91 @@ It may help to understand some typical flows when this Puppet module is
-- [Configuration documentation](README.md#conjur-host-factory)
-
-##### Puppet Configuration
-
-- [Puppet Manifest Configuration](README.md#updating-the-puppet-manifest-1) 
-- [Using Hiera](README.md#using-hiera-1) 
-
-#### Using Windows Registry / Windows Credential Manager Pre-Provisioning
-
-This workflow is described in the
-[Using Windows Registry / Windows Credential Manager (Windows agents only)](README.md#using-windows-registry--windows-credential-manager-windows-agents-only)
-section in the [README.md](README.md) file.
-
-![workflow](diagrams/windows_pre_provision_workflow.png)
-
-Diagram Source: [windows_pre_provision_workflow.txt](diagrams/windows_pre_provision_workflow.txt)
-
-##### References
-
-- [Puppet Forge documentation](https://forge.puppet.com/conjur/conjur#pre-established-host-identity)
-- [Configuration documentation](README.md#using-windows-registry--windows-credential-manager-windows-agents-only)
-
-##### Puppet Configuration
-
-###### Puppet Manifest on Master
-
-File: /etc/puppetlabs/code/environments/production/manifests/site.pp
-
-```
-  node default {
-    include conjur
-  }
-```
-
-###### Windows Puppet Agent Configuration
-
-Conjur connection information and the node's Conjur API key can be configured
-using the PowerShell scripts described in the
-[Using Windows Registry / Windows Credential Manager](README.md#using-windows-registry--windows-credential-manager-windows-agents-only)
-section of the [README.md](README.md) file.
-
-#### Using Host Identity with API Key Configured in Puppet Manifest
+#### Pre-Provisioning Using Host Identity
 
 This workflow is described in the
 [Conjur host identity with API key](README.md#conjur-host-identity-with-api-key)
 section in the [README.md](README.md) file.
 
-##### Initial Use
-
-![workflow](diagrams/host_id_with_api_key_workflow.png)
-
-Diagram Source: [host_id_with_api_key_workflow.txt](diagrams/host_id_with_api_key_workflow.txt)
-
-##### Consequent Use:
-
-![Workflow](diagrams/host_id_with_api_key_workflow2.png)
-
-Diagram Source: [host_id_with_api_key_workflow2.txt](diagrams/host_id_with_api_key_workflow2.txt)
-
-##### References
-
-- [Puppet Forge documentation](https://forge.puppet.com/conjur/conjur#conjur-host-identity-with-api-key)
-
-##### Puppet Configuration
+![workflow](diagrams/pre-provisioned-host-identity.png)
 
-- [Puppet Manifest Configuration](README.md#updating-the-puppet-manifest) 
-- [Using Hiera](README.md#using-hiera) 
+### Running a test environment
 
-### Running a Conjur server locally
-
-Run a preconfigured Conjur instance with `docker-compose up -d`.
-Username is 'admin', password is 'ADmin123!!!!'. The HTTPS endpoint is mapped to port `9443`.
-Once the server is running, view the UI at [localhost:9443/ui](https://localhost:9443/ui).
-You can ignore the cert warning; a self-signed cert is used.
-
-### Running a Puppet master locally
+Run a pre-configured Conjur and Puppet instance by going into the
```

Comment:
Is it generally understood that `Puppet instance` will refer to a Puppet master? Or should it this be `Puppet master`? Same comment for Line 58 below.